Makefile

```diff
@@ -19,18 +19,28 @@ node_modules: package.json package-lock.json
 core/css/uikit.%.css: src/themes/%.less node_modules
 	node_modules/less/bin/lessc src/themes/$*.less core/css/uikit.$*.css --relative-urls
 
+core/css/material-icons.css: src/themes/icons.scss node_modules
+	node node_modules/.bin/sass src/themes/icons.scss core/css/material-icons.css --no-source-map
```

Comment:
is the `node` in front of it required ? I thought one can just use directly whatever is in `.bin` (it might have a shebang)

Reply:
you can replace "core/css/material-icons.css" with `$@` which is an alias for the target rule if you like